main/qml/Window.qml

```diff
@@ -159,20 +148,20 @@ ApplicationWindow {
         }
     }
 
-    property Tab currentTab: mainView.currentTab()
-    
+    property Tab currentTab: mainView.currentTab
+    property View currentView: mainView.currentView
 
     Item {
         anchors.fill: parent
         Keys.onPressed: {
-            var v = view(); if (v === undefined) return;
-            v.ctrl = (event.key == Qt.Key_Control) ? true : false;
+            var v = currentView; if (v === undefined) return;
+            if (event.key == Qt.Key_Control) v.ctrl = true;
             event.accepted = frontend.handleInput(event.text, event.key, event.modifiers)
             event.accepted = true;
```

Comment:
I think it's an attempt to prevent QML from trying to find more things to do with the keypress, like menu items. I don't know if it actually does anything though.